src/components/dev-hub/controlled-tooltip.js

```diff
@@ -76,125 +76,134 @@ const horizontalArrowBase = css`
-    ${rightDefaultArrow}
+const rightGradientArrow = theme => css`
+    ${rightDefaultArrow(theme)}
     &:before {
-        border-left-color: ${colorMap.magenta};
+        border-left-color: ${theme.colorMap.magenta};
     }
     &:after {
-        border-left-color: ${colorMap.greyDarkOne};
+        border-left-color: ${theme.colorMap.greyDarkOne};
     }
 `;
 
-const leftDefaultArrow = css`
+const leftDefaultArrow = theme => css`
     ${arrowBase}
     ${horizontalArrowBase}
     &:before,
     &:after {
         right: 100%;
     }
     &:before {
-        border-right-color: ${colorMap.greyLightTwo};
+        border-right-color: ${theme.colorMap.greyLightTwo};
     }
     &:after {
-        border-right-color: ${colorMap.greyDarkOne};
+        border-right-color: ${theme.colorMap.greyDarkOne};
     }
 `;
 
-const leftGradientArrow = css`
+const leftGradientArrow = theme => css`
     ${leftDefaultArrow}
     &:before {
-        border-right-color: ${colorMap.orange};
+        border-right-color: ${theme.colorMap.orange};
     }
     &:after {
-        border-right-color: ${colorMap.greyDarkOne};
+        border-right-color: ${theme.colorMap.greyDarkOne};
     }
 `;
 
-const bottomDefaultArrow = css`
+const bottomDefaultArrow = theme => css`
     ${arrowBase}
     ${verticalArrowBase}
     &:after {
-        border-top-color: ${colorMap.greyDarkOne};
+        border-top-color: ${theme.colorMap.greyDarkOne};
         top: calc(100% + 11px);
     }
     &:before {
-        border-top-color: ${colorMap.greyLightTwo};
+        border-top-color: ${theme.colorMap.greyLightTwo};
         top: calc(100% + 15px);
     }
 `;
 
-const bottomGradientArrow = css`
+const bottomGradientArrow = theme => css`
     ${bottomDefaultArrow}
     &:after {
-        border-top-color: ${colorMap.greyDarkOne};
+        border-top-color: ${theme.colorMap.greyDarkOne};
     }
     &:before {
-        border-top-color: ${colorMap.magenta};
+        border-top-color: ${theme.colorMap.magenta};
     }
 `;
 
-const topDefaultArrow = css`
+const topDefaultArrow = theme => css`
     ${arrowBase}
     ${verticalArrowBase}
     &:before,
     &:after {
         bottom: 100%;
     }
     &:after {
-        border-bottom-color: ${colorMap.greyDarkOne};
+        border-bottom-color: ${theme.colorMap.greyDarkOne};
     }
     &:before {
-        border-bottom-color: ${colorMap.greyLightTwo};
+        border-bottom-color: ${theme.colorMap.greyLightTwo};
     }
 `;
 
-const topGradientArrow = css`
-    ${topDefaultArrow}
+const topGradientArrow = theme => css`
+    ${topDefaultArrow(theme)}
     &:after {
-        border-bottom-color: ${colorMap.greyDarkOne};
+        border-bottom-color: ${theme.colorMap.greyDarkOne};
     }
     &:before {
-        border-bottom-color: ${colorMap.magenta};
+        border-bottom-color: ${theme.colorMap.magenta};
     }
 `;
 
-const getArrowStyles = (hasGradientBorder, position) => {
+const getArrowStyles = (hasGradientBorder, position, theme) => {
```

Comment:
Something may be missing with this component, it looks like we have this triangle that is usually part of the popover (you can see it by clicking the contents menu on any article) that disappeared:

This branch:
![Screen Shot 2020-06-02 at 11 09 24 AM](https://user-images.githubusercontent.com/9064401/83536959-f0928900-a4c1-11ea-9c2a-0203bb74ec44.png)


Prod:
![Screen Shot 2020-06-02 at 11 09 19 AM](https://user-images.githubusercontent.com/9064401/83536969-f38d7980-a4c1-11ea-95bb-c060467e4629.png)

Reply:
I also just noticed these links should be white on `hover`, looks like here they are green